multimedia/yt-dlp/Makefile

```diff
@@ -0,0 +1,41 @@
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=yt-dlp
+PKG_VERSION:=2022.4.8
+PKG_RELEASE:=1
+
+PYPI_NAME:=yt-dlp
+PKG_HASH:=8758d016509d4574b90fbde975aa70adaef71ed5e7a195141588f6d6945205ba
+
+PKG_MAINTAINER:=Michal Vasilek <michal.vasilek@nic.cz>
+PKG_LICENSE:=Unlicense
+PKG_LICENSE_FILES:=LICENSE
+
+include ../../lang/python/pypi.mk
+include $(INCLUDE_DIR)/package.mk
+include ../../lang/python/python3-package.mk
+
+define Package/yt-dlp
+  SECTION:=multimedia
+  CATEGORY:=Multimedia
+  TITLE:=Utility to download videos from YouTube.com
+  URL:=https://github.com/yt-dlp/yt-dlp
+  DEPENDS:= \
+    +ca-certificates \
+    +python3 \
+    +python3-email \
+    +python3-xml \
+    +python3-codecs \
+    +python3-ctypes \
+    +python3-setuptools
```

Comment:
I removed setuptools because they don't use it and added relevant python3 stdlib dependencies.

Dependencies in distributions are all optional. On my computer, I can download a youtube video with yt-dlp with just python3 installed. If I wanted the best audio and video quality, I would have to install ffmpeg because audio and video files need to be downloaded separately and then combined.

https://github.com/yt-dlp/yt-dlp#dependencies

Reply:
Void Linux (where I maintain yt-dlp) and Alpine Linux only have python in dependencies, Arch and maybe other distributions only have python-certifi (which is also optional) in dependencies.